integration/init/istio-1.0.3/metadata.yaml

```
@@ -0,0 +1,3 @@
+upstream: "https://github.com/istio/istio/tree/1.0.3/install/kubernetes/helm/istio"
+args: []
```

```suggestion
args: ["--prefer-git"]
```
We should include this whenever possible